src/components/EmailForm/index.jsx

```diff
@@ -0,0 +1,77 @@
+import { useState } from "react"
+import { Link } from "react-router-dom"
+
+function EmailForm(){
+    const [email,setEmail]=useState('')
+    const [subject,setSubject]=useState('')
+    const [message,setMessage]=useState('')
```

Comment:
Nice work figuring out the form!

A few things with these state lines, we want spaces after the , and spaces around the =